src/components/input/MultiSelect.stories.ts

```diff
@@ -149,3 +171,41 @@ export const MultipleSelectors: Story = {
     `
   })
 }
+
+export const WithSearchBox: Story = {
+  ...Default,
+  args: {
+    hasSearchBox: true
+  }
+}
+
+export const WithSelectedCount: Story = {
+  ...Default,
+  args: {
+    showSelectedCount: true
+  }
+}
+
```

Comment:
Could you add some default items to these?
I'm lazy and don't like to have to go edit the controls 😅

Reply:
Tried adding plain string options, got 

<img width="718" height="886" alt="Image" src="https://github.com/user-attachments/assets/83714010-d4ea-40cd-8812-46e4fde76e00" />